spotify_player/src/event/mod.rs

```diff
@@ -118,6 +118,14 @@ fn handle_key_event(
         }
     };
 
+    // Example key binding: Alt+P to play the current playlist
+    if &key_sequence.keys[..] == [Key::Alt(crossterm::event::KeyCode::Char('p'))] {
+        client_pub.send(ClientRequest::Player(PlayerRequest::PlayCurrentPlaylist))?;
+        // Mark this event as handled and reset the key sequence
+        ui.input_key_sequence.keys.clear();
+        return Ok(());
+    }
```

Comment:
no this is not a correct pattern. You should define a command with keybindings and handle event based on the command